trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/BET_calculations.py

```diff
@@ -142,8 +142,9 @@ def compute_inflow_and_tip_loss(r,R,Wa,Wt,B):
     lamdaw[lamdaw<0.] = 0.
```

Comment:
this is the line that needs to be fixed:
lamdaw[lamdaw<0.] = 1e-12

Reply:
updated